vuu-ui/packages/vuu-shell/src/theme-switch/ThemeSwitch.tsx

```diff
@@ -26,9 +23,9 @@ export const ThemeSwitch = ({
   className: classNameProp,
   defaultMode: defaultModeProp,
```

Comment:
I think, in the case of this component, we should assume controlled mode only. The ThemeContext is always going to be where this state is managed.